documentation/API_documentation/QoD_API.md

```diff
@@ -308,6 +307,177 @@ Following table defines API endpoints of exposed REST based for QoD management o
+                <b>minDuration (optional)</b>: Minimum duration for a session with this QoS Profile<br>
+                <b>maxDuration (optional)</b>: Maximum duration for a session with this QoS Profile<br>
+                <b>priority (optional)</b>: The priority of this QoS Profile<br>
+                <b>packetDelayBudget (optional)</b>: Maximum latency<br>
+                <b>Jitter (optional)</b>: Jitter refers to the variation in the time it takes for none queue building packets to travel across a network measured in milliseconds (ms). In terms of maximum deviation for round trip latency on a network, jitter can be defined as the difference between the highest and lowest latency values experienced by the 99th percentile of traffic.
+                To be more specific, if we consider the round trip latency values of the 99th percentile of traffic, jitter is the measure of how much these values deviate from the average round trip latency for this percentile. A low jitter value indicates that the latency values are relatively consistent, whereas a high jitter value implies that there is a significant variation in latency times, which can negatively impact the performance of real-time applications such as VoIP, video conferencing, and online gaming.
+                <br>
+                <b>packetErrorLossRate (optional)</b>: The magnitude of the error loss rate<br>
+            </td>
+        </tr>
+        <tr>
+            <td><b>400: Invalid input</b></td>
+        </tr>
+        <tr>
+            <td><b>401: Un-authorized</b></td>
+        </tr>
+        <tr>
+            <td><b>403: Forbidden</b></td>
+        </tr>
+        <tr>
+            <td><b>409: Conflict</b></td>
+        </tr>
+        <tr>
+            <td><b>500: Server Error</b></td>
+        </tr>
+        <tr>
+            <td><b>503: Service temporarily unavailable</b></td>
+        </tr>
+    </tbody>
+</table>
+
+#### QoD - Get QoS Profiles Resource by id
+
+<table>
+    <thead>
+        <tr>
+            <th colspan=3><b>Get QoS Profile Resource by id</b></th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><b>HTTP Request</b></td>
+            <td colspan=2>GET &lt;base-url&gt;/qod/v0/qosProfile</td>
+        </tr>
+        <tr>
+            <td><b>Query Parameters</b></td>
+            <td colspan=2> No Query Parameters</td>
+        </tr>
+        <tr>
+            <td><b>Path Parameters</b></td>
+            <td><b>profile-id</b></td>
+            <td>Extended QoS profile id</td>
+        </tr>
+        <tr>
+            <td><b>Request Body Parameters</b></td>
+            <td colspan=2><b>No Request Body Parameters defined</b></td>
+        </tr>
+        <tr>
+            <td rowspan=7><b>Response</b></td>
+            <td><b>201: QoS Profile created</b></td>
+            <td>
+                Response body:<br>
+                <b>id</b>: The identifier of the QoS Profile<br>
```

Comment:
Definitions below don't match profile definitions in API OAS specification

Reply:
> Definitions below don't match profile definitions in API OAS specification

@eric-murray I thought that we were going to get rid of the redundant docs before the release, so I didn't need to update these.  Did I not follow that discussion? It is early for me and I'm not at 100% during our calls.

Reply:
You can remove these additional documents from the PR if you don't want them to be part of the PR. But I don't see the point in updating them with information that does not match the API definition.

Reply:
I would focus on reviewing the descriptions within the spec, as we are deprecating this other doc, but agree that in that case it is better to keep this file as it was so it doesn't show up in the PR